cogs/utility.py

```diff
@@ -0,0 +1,38 @@
+import discord
+from discord import slash_command
+from discord.ext import commands
+
+
+class Utility(commands.Cog, name="utility"):
+    def __init__(self, bot):
+        self.bot = bot
+
+    @slash_command(brief="Information about the server")
```

Comment:
Please remember to add """ descriptions """. Briefs don't actually add them correctly

Reply:
Are briefs necessary at all right now?

Reply:
Not really.